go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/opencontainers/runc
 
-go 1.20
+go 1.21
```

Comment:
For what function?

Reply:
Yes, no need to do this.
So we need to test  with go 1.20, 1.21 and 1.22 in CI?